guides/user/pipeline/pipeline-templates/index.md

```diff
@@ -34,9 +34,9 @@ on an existing pipeline.
 
 * Share the template with one or more teams of developers using Spinnaker.
 
-  To [save a pipeline
+  [Saving a pipeline
```

Comment:
The rest of the list starts w/ a verb, such as use. Can you rewrite the sentence to be start w/ "Save" so that it's consistent with the others?

Reply:
Hi Brian,

The rest of the list does start with a verb, but I thought this sentence in particular is more of a note to provide additional context to the second bullet. If this sentence should be an action / step, then I think this should appear as the bullet instead. 

Ex.

- Save a pipeline template to Spinnaker.

This will make the template available to developers. It's a good idea to communicate to the team about what templates are available.